src/interface/src/app/login/login.component.html

```diff
@@ -49,22 +49,17 @@ <h1 class="signin-title">Sign in to Planscape</h1>
             message="{{ passwordError }}"
             title="password error"></app-field-alert>
 
-          <div class="login-error-box" *ngIf="loginError">
-            <p class="login-error-heading">
-              <mat-icon class="warning-icon login-error-icon">warning</mat-icon>
-              CREDENTIALS ERROR
-            </p>
-            <p class="login-error-details">
-              {{ loginError }}
-              <a
-                *ngIf="offerReverify"
-                clickable
-                [routerLink]="[]"
-                (click)="resendVerification()"
+          <app-form-message
+            *ngIf="loginError !== '' ? true : false"
```

Comment:
I'm not sure what the best way would be to trigger display of this message, but this seemed like the easiest.